components/browser/state/src/main/java/mozilla/components/browser/state/state/CustomTabSessionState.kt

```diff
@@ -13,13 +13,17 @@ import java.util.UUID
  * @property content the [ContentState] of this custom tab.
  * @property trackingProtection the [TrackingProtectionState] of this custom tab.
  * @property config the [CustomTabConfig] used to create this custom tab.
+ * @property extensionState a maps of web extension ids and extensions that contains all installed web
```

Comment:
It doesn't contain all web extensions, only contains state when there's is a session/tab specific override.

nit: a map[s] of webextensions...

Reply:
Thanks for the catch I really meant the other way around